lib/puppet/type/scheduled_task.rb

```diff
@@ -140,7 +143,7 @@ def insync?(current)
           }
 
           # Run at 8am on the first Monday of the month for January, March, and May,
-          # starting after August 31st, 2011.
+          # starting after August 31st, 2011. Repeat every 10 minutes for 60 minutes.
```

Comment:
I think it might be better to add an additional example that focuses on `minutes_interval` and `minutes_duration`, rather than changing the existing example.